python_rpc/tests/test_install.py

```diff
@@ -0,0 +1,47 @@
+"""Integration test for Python-related Protobuf setup"""
+import subprocess
+import sys
+import tempfile
+import os
+
+def test_pwm_proto_serialization():
+    """Test protobuf installation and correct serialization/deserialization"""
+
+    # Build the package
+    subprocess.check_call([sys.executable, "-m", "pip", "install", "build"])
+    subprocess.check_call([sys.executable, "-m", "build", "python_rpc"])
+
+    # Get the wheel
+    wheel_file = next(
+        (file for file in os.listdir("python_rpc/dist") if file.endswith(".whl")), None
+    )
+    assert wheel_file, "Wheel file not found"
+
+    # Install the package from the wheel
+    with tempfile.TemporaryDirectory() as temp_dir:
+        subprocess.check_call(
+            [
+                sys.executable,
+                "-m",
+                "pip",
+                "install",
+                os.path.join("python_rpc/dist", wheel_file),
+                "--target",
+                temp_dir,
+            ]
+        )
+        sys.path.append(temp_dir)
+
+        # pylint: disable=import-error, wrong-import-position, import-outside-toplevel
+        from protobufs.pwm_pb2 import GetFrequency
```

Comment:
why is this import all the way down here?

Reply:
It's part of the test for checking the installation and import of the protobuf package.